config/index.js

```diff
@@ -1,21 +0,0 @@
-/* eslint-disable global-require, import/no-dynamic-require */
-'ue strict'
-const logger = require('../logger')
-
```

Comment:
you can declare `config` here with let as opposed to creating a global.

Reply:
done I think